student-projects/Elad/class-013/code.ts

```diff
@@ -0,0 +1,61 @@
+const game_level = prompt("welcome to gambling of guess the number, pick a levele of the game:\n 1. easy > 1 - 10\n 2. midlle > 1 - 100\n 3. hard > 1 - 1000  ")
+// here we start level one //
+if (game_level === null || game_level === "") {
+    alert("There seems to be a problem, try refreshing the page.")
+} else {
+    const low_game_level = game_level.toLowerCase();
+    if (low_game_level === "1" || low_game_level === "easy") {
```

Comment:
The structure should be:
1. Prompt for level
2. Validate the level
3. Generate a random number
4. Write the logic game once, when validating check again what is the level